tracekit.d.ts

```diff
@@ -87,13 +91,13 @@ interface Report {
      * Add a crash handler.
      * @param {Function} handler
      */
-    subscribe(handler:(stackTrace:StackTrace, options?:any) => void): void;
+    subscribe(handler: ReportSubscriber): void;
```

Comment:
Do you know what options was before, looking below it looks like it could be all the arguments. I think it's the same, but just want to confirm.